specification/azurelargeinstance/AzureLargeInstance.Management/models.tsp

```diff
@@ -292,20 +292,44 @@ model AzureLargeInstanceProperties {
   @doc("Specifies the network settings for the Azure Large Instance.")
   networkProfile?: NetworkProfile;
 
+  @removed(Versions.v2024_08_01_preview)
+  @renamedFrom(Versions.v2024_08_01_preview, "azureLargeInstanceId")
   @doc("Specifies the Azure Large Instance unique ID.")
   @visibility("read")
+  azureLargeInstanceIdOld?: string;
+
```

Comment:
I'm not familiar with this pattern. What is this for?

Reply:
This is a [workaround](https://azure.github.io/typespec-azure/docs/next/troubleshoot/versioning_operation_template_changes) in typespec that's needed to update the visibility / decorator of a variable.